app/models/Search.java

```diff
@@ -487,4 +495,15 @@ private SearchResponse search(final QueryBuilder queryBuilder,
 	private void setMessageOut(final Chunks.Out<String> messageOut) {
 		this.messageOut = messageOut;
 	}
+
+	/**
+	 * As there is a memory issue within Promise we musn't allow data > RAM. As
+	 * experience shows, this is around 4 M docs (dependig on the serialisation).
+	 * 
+	 * @return maximum of allowed hits for a scroll search
+	 */
+
+	public static long getAllowedMaxScrollHits() {
+		return maxScrollHits;
```

Comment:
Yeah thats what I did first, than eclipse copmplaining about missing description resp. suggesting to use getter. I like the constant more, too, so I will follow your suggestion.